src/_playground/Routes.js

```diff
@@ -327,10 +341,12 @@ export class Routes extends Component {
             );
         });
 
+        const showNav = this.state.showSideNav ? '' : 'sidebar-hidden';
+
         return (
             <BrowserRouter basename={process.env.PUBLIC_URL}>
                 <div className='frDocs-Container'>
-                    <div className='frDocs-Sidebar'>
+                    <div className={`frDocs-Sidebar ${showNav}`}>
```

Comment:
A cleaner way of implementing conditional CSS classes is to use the `classnames` package.  You can check a number of other components for usage examples.

Reply:
great, i'll take a look 👍